apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryClient.JobCrud.cs

```diff
@@ -295,13 +378,101 @@ public virtual Task<BigQueryJob> GetJobAsync(JobReference jobReference, GetJobOp
         /// Asynchronously cancels a job.
         /// </summary>
         /// <param name="jobReference">A fully-qualified identifier for the job. Must not be null.</param>
-        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied. Must not be null.</param>
```

Comment:
It doesn't look like this should have been added. Directly contradicts the previous sentence.